core/foundation/src/TClassEdit.cxx

```diff
@@ -1134,8 +1134,13 @@ int TClassEdit::GetSplit(const char *type, vector<string>& output, int &nestedLo
 
       const char *cursor;
       int level = 0;
+      int parenthesis = 0;
       for(cursor = c + 1; *cursor != '\0' && !(level==0 && *cursor == '>'); ++cursor) {
+         if (parenthesis && *cursor != ')')
```

Comment:
Can `parenthesis` ever become `> 1`? Else consider making it a `bool`.

Reply:
I suppose it could, the function could itself as parameters that are functions ....

Reply:
but you are right the code is wrong/inconsistent ;(